### PR TITLE
feat(APIChannel): channel banners

### PR DESCRIPTION
**Pull request by suneettipirneni**
**Please describe the changes this PR makes and why it should be merged:**

**Reference Discord API Docs PRs or commits:**
- https://github.com/discord/discord-api-docs/pull/4068

### CODE REVIEW

File: deno/payloads/v8/guild.ts [hunk 1]
```diff
@@ -513,6 +513,10 @@ export enum GuildFeature {
 	 * Guild has enabled the welcome screen
 	 */
 	WelcomeScreenEnabled = 'WELCOME_SCREEN_ENABLED',
+	/**
+	 * Guild has access to set a channel banner
+	 */
+	ChannelBanner = 'CHANNEL_BANNER',
 }
```
Comment (vladfrangu) [inline deno/payloads/v8/guild.ts:520]: Please sort features alphabetically

**Review by vladfrangu — CHANGES_REQUESTED** [review]

**Comment by vladfrangu** [comment]
This needs a rebaseeeee 🎵

**Review by vladfrangu — APPROVED** [review]
Make sure rest types include these

**Comment by advaith1** [comment]
Channel banners were removed and guilds no longer have the feature.

**Pull request closed without merging**